index.html

```diff
@@ -0,0 +1,207 @@
+            <i class="fas fa-toolbox"></i>
+          </div>
+
+          <div class="program__title">
+            <h3>Diversity gap</h3>
+          </div>
+
+          <div class="program__description">
+            <p>
+              Has Covid changed the status quo for women and/or ethnic
+              minorities in the workplace?
+            </p>
+          </div>
+        </div>
+
+        <!-- program 3 -->
+        <div class="program_list">
+          <div class="program__icon">
+            <i class="fas fa-users"></i>
+          </div>
+
+          <div class="program__title">
+            <h3>Social structure</h3>
+          </div>
+
+          <div class="program__description">
+            <p>
+              Remote working, at least in some form, is likely here to stay for
+              businesses across nearly every sector.
+            </p>
+          </div>
+        </div>
+
+        <!-- program 4 -->
+        <div class="program_list">
+          <div class="program__icon">
+            <i class="fas fa-recycle"></i>
+          </div>
+
+          <div class="program__title">
+            <h3>Recreation</h3>
+          </div>
+
+          <div class="program__description">
+            <p>
+              What will work look like if our offices become virtual? How can
+              tech step up to boost workplace social interactions?
+            </p>
+          </div>
+        </div>
+      </div>
+
+      <a class="link" href="#">SEE THE WHOLE PROGRAM</a>
+      <div class="btn btn--primary m-2">
+        <a href="#">Join CC Global summit 2022</a>
+      </div>
+    </section>
+
+    <!-- ---------- Speaker Section --------- -->
+    <section class="speaker_section container">
+      <h2>Feature Speaker</h2>
+      <div class="line"></div>
+
+      <div class="speaker_list"></div>
+    </section>
+
+    <!-- ---------- partner Section --------- -->
+    <section class="partner_section container">
+      <h2>Partners</h2>
+      <div class="line"></div>
+
+      <div class="partners">
+        <img src="./images/partner/google.png" alt="google" class="" />
+        <img src="./images/partner/amazon.png" alt="amazon" class="" />
+        <img src="./images/partner/meta.png" alt="meta" class="" />
+        <img src="./images/partner/microsoft.png" alt="Ms" class="" />
+        <img src="./images/partner/netflix.png" alt="netflix" class="" />
+      </div>
+    </section>
+
+    <!-- ---------- Footer Section --------- -->
+    <footer class="footer_section container">
+      <img src="./images/logo.png" alt="logo brand" />
+
+      <div class="content">
+        <p>2022 Creative commons british<br />some rights reserved</p>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Distinctio
+          minima quas quidem aut eius aperiam nisi totam dicta itaque iste modi
+          quaerat, esse saepe voluptatum natus rem reiciendis. Molestiae, id.
+        </p>
+      </div>
+    </footer>

```

Comment:
- [ ] I recommend decreasing the size of logo in mobile screens

Your design  | Original design
------------- | ------------- 
<img width="384" alt="Screenshot 2022-06-23 at 22 24 01" src="https://user-images.githubusercontent.com/77038610/175368473-8607663c-1005-43ff-815d-e45ee2f68bc9.png"  >| <img width="463" alt="Screenshot 2022-06-23 at 22 23 35" src="https://user-images.githubusercontent.com/77038610/175368388-8b02e9cb-4f0b-426e-ad62-ffac65892dde.png">

- [ ] In desktop size, About page's footer  should have  a dark background. Please delete partners section from desktop size (About page)

Your design  | Original design
------------- | ------------- 
<img width="772" alt="Screenshot 2022-06-23 at 22 26 03" src="https://user-images.githubusercontent.com/77038610/175368820-9e7edd69-bcd5-4a7d-88ab-f5d26e34c2ff.png"> | <img width="857" alt="Screenshot 2022-06-23 at 22 26 22" src="https://user-images.githubusercontent.com/77038610/175368861-68c2068d-5e0e-4af1-b915-80987b97f3e9.png">

Reply:
I decreased the size of the log. even this is not a very big deal because you resized my project very small size but the right original screenshot is much wider than the left. 

but yes you are right about deleting the partner section, I did not notice that, also changing the color of the footer. (Fixed)